### ISSUE TITLE
Suggestion: add a copy button for the correct bank character to the existing copy buttons

### ISSUE THREAD

**Issue by TheAmazingTyr** [Icecrown-Community/ICC-Member-Website-Public#62]

**Comment by Dejnaa** [comment]
Or perhaps it would be better in the long run if the info told you what realm to log on (we have bankalt on that realm). I find that I often have to open Icc Bankalt sheet to check what a realm is connected to and what realm we have the bankalt on. I have 5 accounts, and only make alts on the realms that we have banks on, the other realms are not needed? :P

**Comment by Bgabor997** [comment]
Like displaying a warning  icon in the booking card, with a tooltip what telling you, like: " We don't have bank character on Mal'Ganis, log in to Blackhand"?

**Comment by TheAmazingTyr** [comment]
i think the idea is to have a third line, as we have - customer realm, paid realm, and then add the third one, bank realm
while i agree, it'll then go back into needing to copy both the customer name AND the realm, so for that, #58  would be required to be running, and there's a question of what GCs care about more, duplicate chars on merged realms or potential inconvenience in inviting/whispering

**Comment by Dejnaa** [comment]
The list of realms do not update that often, so if you could put the data in 1 time, it could display the "correct realm" next to "payment realm". For example:  (only in GC view)

![image](https://user-images.githubusercontent.com/77668716/120908111-74e49280-c667-11eb-9c4a-a88b8798370b.png)

**Comment by iElsha** [comment]
Waiting for Precious bank character feature

**Comment by Dejnaa** [comment]
Do we really need this now when we got Zazus Addon?

**Comment by Dejnaa** [comment]
@Tueri @ICCDesi

What do you think?

**Comment by Bgabor997** [comment]
I know its not me who u pinged :D but as a gc i don't think this is needed

**Comment by Dejnaa** [comment]
I agree, just wanted to see if the Big Heads saw any good in it before I asked that someone closes the ticket ^^

**Comment by Efthymios-Papakostopoulos** [comment]
#109
